e2e/rbd_helper.go

```diff
@@ -424,6 +430,16 @@ func validateEncryptedPVC(f *framework.Framework, pvc *v1.PersistentVolumeClaim,
 	return validateEncryptedImage(f, rbdImageSpec, app)
 }
 
+func isThickPVC(f *framework.Framework, pvc *v1.PersistentVolumeClaim, app *v1.Pod) error {
```

Comment:
@nixpanic may be a quick source comment about this functions can be added for both `isThickPVC and isEncryptedPVC`

Reply:
I do not think that is needed, the names of the functions are pretty self explaining.